book/plugins/general-abilities/multiple-types-one-repo/multiple-types-one-repo.md

```diff
@@ -0,0 +1,15 @@
+# One Code Base for Multiple Plugins
+
+A developer can open one repository for handling different plugin types.
+
+For example, you can build a push plugin and later add support for an analytic plugin in the same repository.
+
+The key idea for managing such a scenario is "separation of concerns", each plugin type should be implemented on a different class.
```

Comment:
replace the `,` with `.` and capitalize after.
Those are 2 separate sentences

Reply:
Done